core/src/i18n/en.yaml

```
@@ -18,8 +21,26 @@ cluster-role-bindings:
 cluster-roles:
   title: Cluster Roles
 clusters:
+  list:
+    outdated:
+      download: Download Kubeconfig
+      tooltip: Outdated parameter version may cause errors. Delete and re-add your cluster.
   overview:
     title: Cluster Overview
+common:
+  delete-dialog:
+    buttons:
+      cancel: Cancel
+      delete: Delete
+    delete-confirm: Don't show delete confirmations
+    message: Are you sure you want to delete {{type}} {{name}}?
```

```suggestion
    message: Are you sure you want to remove {{type}} {{name}}?
```

consistency with title